Tests/Integration/Components/ConnectExportTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests\Integration\Components;
+
+use Shopware\Components\Model\ModelManager;
+use ShopwarePlugins\Connect\Components\ConfigFactory;
+use ShopwarePlugins\Connect\Components\ConnectExport;
+use ShopwarePlugins\Connect\Components\ErrorHandler;
+use ShopwarePlugins\Connect\Components\Validator\ProductAttributesValidator\ProductsAttributesValidator;
+use ShopwarePlugins\Connect\Tests\DatabaseTestCaseTrait;
+use Shopware\Models\Category\Category;
+use Tests\ShopwarePlugins\Connect\ConnectTestHelper;
+
+class ConnectExportTest extends ConnectTestHelper
```

Comment:
ConnectTestHelper is not used anymore in new tests structure. So it's not possible integration test to extend it.

Reply:
why not?
it is possible and i use things from it to get the SDK-instance and a helper instance.
Alternative 1: still extend ConnectTestHelper if it's necessary
Alternative 2: create a trait for getting those things
Alternative 3: copy code from ConnectTestHelper into each testcase

Reply:
I saw that it's used also in other places as well. If I remember correctly we decided together with Simon that we'll use traits instead of this ConnectTestHelper.